roles/ceph-dashboard/tasks/configure_dashboard_backends.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: get current mgr backend - ipv4
+  set_fact:
+      mgr_server_addr: "{{ hostvars[dashboard_backend]['ansible_all_ipv4_addresses'] | ips_in_ranges(public_network.split(',')) | first }}"
```

Comment:
minor thing but we're using 2 spaces indent instead of 4

Reply:
+1